src/main/resources/db/migration/V4__Create_Published_Tables.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE conceptdata ADD COLUMN revision integer not null default 1;
```

Comment:
Eg liker alltid å ha med if not exists for alle slike migreringer. Då kan den kjøres fleire gonger uten at det gjør noko.
```suggestion
ALTER TABLE conceptdata ADD COLUMN IF NOT EXISTS revision integer not null default 1;
```

Reply:
:+1: